### PR TITLE
Fix additional information links in translation pages

### PR DESCRIPTION
**Pull request by rodrigowebjump**
## Purpose of this pull request

This pull request (PR) fix additional information links in translation pages

## Affected DevDocs pages
https://devdocs.magento.com/guides/v2.2/config-guide/cli/config-cli-subcommands-i18n.html
https://devdocs.magento.com/guides/v2.2/frontend-dev-guide/translations/xlate.html
https://devdocs.magento.com/guides/v2.2/frontend-dev-guide/translations/theme_dictionary.html
https://devdocs.magento.com/guides/v2.2/frontend-dev-guide/translations/translate_theory.html
https://devdocs.magento.com/guides/v2.2/frontend-dev-guide/translations/translate_practice.html

https://devdocs.magento.com/guides/v2.3/config-guide/cli/config-cli-subcommands-i18n.html
https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/translations/xlate.html
https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/translations/theme_dictionary.html
https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/translations/translate_theory.html
https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/translations/translate_practice.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.2/config-guide/cli/config-cli-subcommands-i18n.md [hunk 1]
```diff
@@ -323,11 +323,9 @@ Similar to the preceding example, generate a `.csv` file, but instead of specify
 
 ## Additional information
 
--  [Translations overview]
--  [Translate theme strings]
+-  [Translations]
```
Comment (keharper) [inline src/guides/v2.2/config-guide/cli/config-cli-subcommands-i18n.md]: Back out the changes in this file.  There's no reason to delete the link to "Translate theme strings", and the title of the `xlate.md` is "Translations overview"

File: src/guides/v2.2/frontend-dev-guide/translations/theme_dictionary.md [hunk 2]
```diff
@@ -76,11 +76,17 @@ You can translate the same element in different ways for different modules:
 ## Additional information
 
 -  [Translations overview]
+-  [Translate theme strings]
```
Comment (keharper) [inline src/guides/v2.2/frontend-dev-guide/translations/theme_dictionary.md]: Don't add links to topics that are in the same section of the left navigation. Remove those links, but keep the one to the translation project.

File: src/guides/v2.2/frontend-dev-guide/translations/translate_practice.md [hunk 3]
```diff
@@ -52,13 +52,17 @@ For example:
 ## Additional information
 
 -  [Translations overview]
--  [Translation dictionaries and language packages]
--  [Usetranslation dictionary to customize strings]
+-  [Use translation dictionary to customize strings]
```
Comment (keharper) [inline src/guides/v2.2/frontend-dev-guide/translations/translate_practice.md]: Same comment as above.

**Comment by keharper** [comment]
I forgot to tag @rodrigowebjump when I left my review.

**Comment by jeff-matthews** [comment]
If the peer review instructions are to back out changes, then this PR becomes invalid.

**Pull request closed without merging**